src/components/Input/input.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import "./input.css";
+
+const Input = ({ updateTask, printInput }) => {
```

Comment:
const Input ? what do you mean by Input? @PuniaMohit  use better naming conventions, think & write